### PR TITLE
Change/educate

### PR DESCRIPTION
**Pull request by luisahorsten**

### CODE REVIEW

**Comment by FelixHenninger** [comment]
Thanks! This looks good, but there's some kind of syntax error in the link to `researchtransparency.org`. Might I ask you to double-check please?

Thanks, -Felix

File: content/change/educate/literature.md [hunk 1]
```diff
@@ -1,15 +1,22 @@
 title: Introduction
 
-# Where can I learn more about the issues?
+After more than half a decade of discussion, the literature on reproducible science is substantial. Thankfully, brave authors have provided summaries and reviews. If you're looking for a place to get started, search no more, we've got you covered!
```
Comment (FelixHenninger) [inline content/change/educate/literature.md]: Yeees!

File: content/change/educate/literature.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
 
-After more than half a decade of discussion, the literature on reproducibility is substantial. Thankfully, brave authors have provided summaries and reviews:
 
-* [Spellman, Gilbert, & Corker \(2017\)](https://psyarxiv.com/ak6jr) provide practical, actionable advice for **Open Science: What, Why, and How**
+# What started the Open Science Movement in the first place and why is it so important?
```
Comment (FelixHenninger) [inline content/change/educate/literature.md]: Would it make sense to nest this heading?

File: content/change/educate/first-steps.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
+title: First steps
```
Comment (FelixHenninger) [inline content/change/educate/first-steps.md]: I think these are no longer strictly necessary, as page titles are set in the `mkdocs.yaml` file in the new `mkdocs` version.

File: content/change/educate/literature.md [hunk 4]
```diff
@@ -1,15 +1,22 @@
 title: Introduction
```
Comment (FelixHenninger) [inline content/change/educate/literature.md]: You could cut this while you're at it, but it's optional.

File: content/change/get_started.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+title: Start the Change
```
Comment (FelixHenninger) [inline content/change/get_started.md]: I like this page very much -- could you make sure to include it in `mkdocs.yml`, and possibly remove the title attribute?

File: content/change/educate/literature.md [hunk 6]
```diff
@@ -1,15 +1,22 @@
...
-* The [Berkeley Initiative for Transparency in the Social Sciences (BITSS)
- ](https://www.bitss.org/) provides a five-week online course on [**Transparent and Open Social Science Research**](https://www.futurelearn.com/courses/open-social-science-research) that covers every step of the research process. Similarly, the [**Open Science MOOC**](https://opensciencemooc.eu/) is developing [course material](https://docs.google.com/document/d/1KuTSECSYHXZmZX15GDjyD65pJ90eRMhHVEZ-1trsw30/edit#) for learning about Open Science, and has collected many interesting links for different topics that often go beyond scholarly articles.
-* The [**Open Science Training Handbook**](https://book.fosteropenscience.eu) is a fantastic and comprehensive resource that covers many topics around open science. It's built for people who would like to create open science training, and very in-depth.
+
+* [Spellman, Gilbert, & Corker \(2017\)](https://psyarxiv.com/ak6jr) provide practical, actionable advice for **Open Science: What, Why, and How**
```
Comment (FelixHenninger) [inline content/change/educate/literature.md]: Do you like Nelson et al. better than Spellman et al.?

**Comment by FelixHenninger** [comment]
Otherwise, this looks truly excellent, looking forward to merging! Left a few comments in the file for your consideration, but I'm nitpicking. Thanks for your work on this.

File: content/change/educate/first-steps.md (hunk 3 above)
Comment (luisahorsten) [reply content/change/educate/first-steps.md]: a'ight!

**Comment by luisahorsten** [comment]
> Thanks! This looks good, but there's some kind of syntax error in the link to `researchtransparency.org`. Might I ask you to double-check please?

Yes! I just learned that "http://" is vital for links on here...

**Comment by FelixHenninger** [comment]
Included in ba96072148e43c89be47f2fa31904f9d3bea7ec2

**Pull request closed without merging**